jetty-server/src/main/java/org/eclipse/jetty/server/ProxyConnectionFactory.java

```diff
@@ -620,6 +620,11 @@ public InetSocketAddress getRemoteAddress()
             return _remote;
         }
 
+        @Override
+        public String toString() {
+            return _endp.toString();
```

Comment:
I don't think this is right, as it will use the `_endp` local and remote addresses, while we want the proxied ones.